docs/guide/index.md

```diff
@@ -44,7 +44,7 @@ pnpm add @types/three -D
 
 ## Vite
 
-If you are using Vite, you have add the following to your `vite.config.ts`:
+If you are using Vite, you just need to add this to your `vite.config.ts` inside of the vue plugin:
```

Comment:
Here I am unsure, the wording on the installation and the index page is slightly different - maybe it makes sense to use the same wording on both pages?

Reply:
I added the same sentence to both getting started and index page in english - hope that works for you guys